package.nls.json

```diff
@@ -80,7 +80,7 @@
     "AWS.configuration.description.awssam.debug.debuggingPlatform": "The debugger to be used with the Image-based Lambda Function",
     "AWS.configuration.description.awssam.debug.timeout": "The amount of time (in seconds) that Lambda allows a function to run before stopping it.",
     "AWS.configuration.description.awssam.debug.aws": "AWS connection details",
-    "AWS.configuration.description.awssam.debug.credentials": "The AWS credential type and ID to use during the invocation. Example: credential profile \"default\" would be entered as `profile:default`.",
+    "AWS.configuration.description.awssam.debug.credentials": "The AWS credential source and name to use during the invocation. Example: credential profile \"default\" would be entered as `profile:default`.",
```

Comment:
I'm not sure what the correct wording here is but I'm not sure "credential source" is the correct one. Credential source already has a meaning in the context of assuming an IAM role: https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#using-aws-iam-roles .

For clarity's sake, it may be worth converting the description in package.json to a `markdownDescription` and providing a link to the docs via hyperlink in the localized string.

FWIW I don't care all that much what it's called through the codebase, but this is a user-facing docstring and I want to ensure this doesn't sow confusion.

Reply:
How about

- "credentials configuration"
- "credentials device"
- "credentials shape"


> For clarity's sake, it may be worth converting the description in package.json to a `markdownDescription` and providing a link to the docs via hyperlink in the localized string.

would be good, but which docs? this `thing:profile` pair we've come up with is not in the docs, is it?

Reply:
I'm assuming this is a concept we've come up with, so I'm thinking we should consider writing docs for this ourselves.

Reply:
Ok, is that blocking this PR?